packages/pigment-css-react/README.md

```diff
@@ -663,6 +663,99 @@ declare module '@pigment-css/react/theme' {
 }
 ```
 
+## Right-to-left support
+
+To support right-to-left (RTL) languages, add the `dir="rtl"` attribute to your app's `<html>` element or any other equivalent top level container. Then, update your bundler config as follows:
+
+### Next.js
+
+```js
+const { withPigment } = require('@pigment-css/nextjs-plugin');
+
+// ...
+module.exports = withPigment(nextConfig, {
+  theme: yourCustomTheme,
+  // CSS output option
+  css: {
+    // Specify your default CSS authoring direction
+    defaultDirection: 'ltr',
+    // To output CSS for the opposite direction to `defaultDirection`
+    // Default is `false`.
+    generateForBothDir: true,
+  },
+});
+```
+
+### Vite
+
+```js
+import { pigment } from '@pigment-css/vite-plugin';
+
+export default defineConfig({
+  plugins: [
+    pigment({
+      theme: yourTheme,
+      css: {
+        // Specify your default CSS authoring direction
+        defaultDirection: 'ltr',
+        // To output CSS for the opposite direction to `defaultDirection`
+        // Default is `false`.
+        generateForBothDir: true,
+      },
+    }),
+    // ... other plugins.
+  ],
+});
+```
+
+### Generated CSS
+
+For example, if you've specified `defaultDirection: 'ltr'` and `dir="rtl"`, and your authored CSS looks like this:
+
+```js
+import { css } from '@pigment-css/react';
+
+const className = css`
+  margin-left: 10px,
+  margin-right: 20px,
+  padding: '0 10px 20px 30px'
+`;
+```
+
+Then the actual CSS output would be:
+
+```css
+.cmip3v5 {
+  margin-left: 10px;
+  margin-right: 20px;
+  padding: 0 10px 20px 30px;
+}
+
+[dir='rtl'] .cmip3v5 {
+  margin-right: 10px;
+  margin-left: 20px;
+  padding: 0 30px 20px 10px;
+}
+```
+
+### Custom dir selector
+
+The default selector in the output CSS is `[dir=rtl]` or `[dir=ltr]`. You can customize it by passing an optional `getDirSelector` method to the `css` property in your bundler config:
+
+```js
+  plugins: [
+    pigment({
+      theme: yourTheme,
+      css: {
+        getDirSelector(dir: string) {
+          // return a custom selector you'd like to use
+          return `:dir(${dir})`;
+        },
+      },
```

Comment:
Everything else looks good but this particular piece is syntax for vite config. Maybe we can just focus from the `css: {` line.